packages/mdc-select/index.js

```diff
@@ -54,6 +56,12 @@ export class MDCSelect extends MDCComponent {
     this.foundation_.setDisabled(disabled);
   }
 
+  constructor(...args) {
+    super(...args);
+
+    this.ripple;
```

Comment:
What's this statement for?

Reply:
Oooh, that's some trash left over from a previous commit. Removing.